### PR TITLE
 Fix Identity Facade Edge

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- Adding Sackmesser cache config deployment
- Reorder imports in package.json

**Fixes:** #279 #280 #281

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Review by JoelGauci — APPROVED** [review]

File: references/identity-facade/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ connect to the IdP
 
 ## Dependencies
 
-- [Maven](https://maven.apache.org/)
+- [Maven (Version >= 3.6.1)](https://maven.apache.org/)
```
Comment (laughingbiscuit) [inline references/identity-facade/README.md]: aim for 3.6.0 if possible for compatibility with debian repos

**Review by laughingbiscuit — APPROVED** [review]

File: references/identity-facade/README.md (hunk 1 above)
Comment (danistrebel) [reply references/identity-facade/README.md]: ACK removed ntp from sackmesser for now

**Pull request merged**